### ISSUE TITLE
Incorrect/False positive

### ISSUE THREAD

**Issue by CodeCruncher24** [hagezi/dns-blocklists#4133]
### Which AdBlocker/DNS cloud service do you use?

ControlD

### Other

https://www.brightedge.com

### ControlD users

- [X] IMPORTANT - I assure that I have not set the Block Response in ControlD to Custom or Branded and can confirm that the problem still occurs.

### NextDNS users

- [ ] IMPORTANT - I can assure that I disabled the block page in NextDNS and can confirm that the problem still occurs.

### With which block list(s) does the problem occur?

Multi LIGHT

### Which domain(s) should be unblocked?

https://www.brightedge.com
https://www.brightedge.com/perplexity

### Why should the domain(s) be unblocked?

It seems like a false positive or unless it has some trackers which need to be blocked but not the whole domain as it seems legit and informative. 
It opens on cloudflare and nextdns

### Privacy

- [X] I confirm that the report does not contain any private information.

**Comment by CodeCruncher24** [comment]
Is also opening on OISD - Full (control D)

**Comment by CodeCruncher24** [comment]
Hi there, can you pls confirm if blocking links from emails which redirect to notion is a false positive or actually blocking major trackers? This is happening on light list.

**Comment by CodeCruncher24** [comment]
Seems to open fine with 1hosts lite filter while blocking the actual tracking perimeter.

**Comment by hagezi** [comment]
Please do not post in closed issues if it does not concern the issue itself.

e.customeriomail.com is not blocked, it is a referral domain.

**Comment by CodeCruncher24** [comment]
Sorry about that. However using light lists, it is not opening so might need to add to whitelist manually.

**Comment by hagezi** [comment]
Then open a new issue, give me details of what exactly it is about. Preferably a sample link to test ...
